### PR TITLE
Medical - Add condition for surgicial kit

### PR DESCRIPTION
**Pull request by Katalam**
**When merged this pull request will:**
- Add fnc_canStitch
- Added condition in Medical Actions
- #6913

### CODE REVIEW

File: addons/medical_treatment/functions/fnc_canStitch.sqf [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#include "script_component.hpp"
+/*
+ * Author: Katalam
+ * Prevents stitch actions from showing if the body is either fully sitched or has no open wounds.
+ *
+ * Arguments:
+ * 0: The medic <OBJECT>
+ * 1: The patient <OBJECT>
+ * 2: Body part <STRING>
```
Comment (TheMagnetar) [inline addons/medical_treatment/functions/fnc_canStitch.sqf]: Parameters 2 and 3 are not used

File: addons/medical_treatment/functions/fnc_canStitch.sqf [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ *
+ * ReturnValue:
+ * Can Stitch <BOOL>
+ *
+ * Public: No
+ */
+
+params ["", "_patient"];
+
+if ((count (_patient getVariable [QEGVAR(medical,bandagedWounds), []])) > 0) exitWith {true};
```
Comment (TheMagnetar) [inline addons/medical_treatment/functions/fnc_canStitch.sqf]: `isEqualTo []` is faster than count

also you can substitute everything and spare conditionals and returns by making it a single line

```
params ["", "_patient"];

!((_patient getVariable [QEGVAR(medical,bandagedWounds), []]) isEqualTo [])

```

File: addons/medical_treatment/functions/fnc_canStitch.sqf [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ * 0: The medic <OBJECT>
+ * 1: The patient <OBJECT>
+ * 2: Body part <STRING>
+ * 3: Treatment class name <STRING>
+ *
+ * ReturnValue:
+ * Can Stitch <BOOL>
+ *
+ * Public: No
+ */
```
Comment (TheMagnetar) [inline addons/medical_treatment/functions/fnc_canStitch.sqf]: Example is missing. Each header has to follow the same rules. For example: https://github.com/acemod/ACE3/blob/master/addons/common/functions/fnc_canDig.sqf

File: addons/medical_treatment/functions/fnc_canStitch.sqf [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#include "script_component.hpp"
+/*
+ * Author: Katalam
+ * Prevents stitch actions from showing if the body is either fully sitched or has no open wounds.
```
Comment (dedmen) [inline addons/medical_treatment/functions/fnc_canStitch.sqf]:
```suggestion
 * Prevents stitch actions from showing if the body is either fully stitched or has no open wounds.
```

**Pull request merged**